File: examples/sample1-python3/judger/judger.py
#!/usr/bin/python3

import os
import sys
import random
import time

playerNum = int(input())
path = input()
with open(path + "fk.txt", 'w') as out:
    out.write("start\n")
    
    out.write("{}\n{}\n".format(playerNum, path))

    print("#send 1")
    print("1")
    print("#continue 1")
    sys.stdout.flush()

    pair = input().split()
    p = int(pair[0])
    ret = str(pair[1])
    print("#send 2")
    print("2")
    print("#continue 2")
    sys.stdout.flush()
    
    print("#log")
    print("test log")
    sys.stdout.flush()

    pair = input().split()
    p = int(pair[0])
    ret = str(pair[1])
    print("#win 1")
    print("1 0")
    print("MLG pro from python")
    sys.stdout.flush()

File: examples/sample1/judger/judger.cpp
#include <iostream>
#include <iomanip>
#include <cstdlib>
#include <ctime>
#include <string>
#include <fstream>

using namespace std;

int main() {
    srand(time(nullptr));
    std::string path;
    int playerNum;
    int currentPlayer = 1;
    
    cin >> playerNum >> path;
    // open saved data
    ofstream out((path + "fk.txt").c_str());

    out << playerNum << endl;
    out << path << endl;
    
    cout << "#send 1" << endl;
    cout << "1" << endl;
    cout << "#continue 1" << endl;

    int p;
    string ret;
    
    cin >> p >> ret;

    cout << "#send 2" << endl;
    cout << "2" << endl;
    cout << "#continue 2" << endl;
    
    cout << "#log" << endl;
    cout << "test log" << endl;
    
    cin >> p >> ret;
    
    cout << "#win 1" << endl;
    cout << "1 0" << endl; // scores
    cout << "MLG pro" << endl; // comments (optional)

    return 0;
}
